lms/static/js/sysadmin/mgmt_commands.js

```diff
@@ -130,13 +152,13 @@
             },
             error: function(std_ajax_err) {
               console.log('Management Command failed to execute');
-            }
+            },
```

Comment:
I know we spoke about standardizing, but I wasn't expected you to update these retroactively.

As @caesar2164 mentioned, it's easier to parse changes if these are kept in a separate commit.